projects/components/src/lib/inline-message/inline-message.service.ts

```diff
@@ -0,0 +1,22 @@
+import {Injectable} from '@angular/core';
+
+import {BehaviorSubject, Observable} from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
```

Comment:
The way it is now, having multiple inline message components on a page would all get triggered at the same time when the user only intends to open one. I'd suggest controlling the title and body component directly in the parent component, skipping having a service altogether to reduce complexity.